docker/Dockerfile

```diff
@@ -54,7 +54,6 @@ RUN chmod +x /entrypoint.sh && \
 ARG ENV_CUDA_PATH=""
 ENV CUDA_PATH=${ENV_CUDA_PATH}
 ENV IN_DOCKER=True
-ENTRYPOINT [ "/entrypoint.sh" ]
```

Comment:
this will help development, hopefully the reasons it was added in the first place don't apply